components/ord-service/src/main/java/com/sap/cloud/cmp/ord/service/repository/ApiSpecRepository.java

```diff
@@ -1,11 +1,14 @@
 package com.sap.cloud.cmp.ord.service.repository;
 
+import java.util.List;
```

Comment:
List seems unused, also you can revisit all the files for unused imports since it is a common issue when developing Java.